Cargo.toml

```diff
@@ -10,11 +10,10 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/weareseba/bdk-reserves"
 
 [dependencies]
-bdk = { version = "0.19", default-features = false }
+bdk = { version = "0.20", default-features = false }
 bitcoinconsensus = "0.19.0-3"
 log = "^0.4"
 
 [dev-dependencies]
 rstest = "^0.11"
 bdk-testutils = "^0.4"
-bdk = { version = "0.19", default-features = true }
```

Comment:
I think this line needs to stay.. it uses electrum while running integration tests..